### DISCUSSION TITLE
@formily/element能否加上CodeSandbox入口

### DISCUSSION THREAD

**Discussion by BlueYeah** [alibaba/formily#2145] [Q&A 2.x]
请问能否给@formily/element的文档例子内加上CodeSandbox入口,有组件问题想自己先排查一下。

例如:
使用@formily/element的form-layout，不能正常使用layout的inline布局。
同样的JSONSchmea而我用@formily/antd 是正常的。

谢谢！

**Comment by muuyao** [comment]
目前还没加，可以先用这个模板 https://codesandbox.io/s/github/Justineo/vue-demi-vue-2-template ，欢迎提pr
